bin/main.ml

```diff
@@ -220,6 +221,80 @@ let check dir =
   OCaml_hashes.check dir;
   Nodes.check dir
 
+let check_int msg expected got =
+  if expected <> got then (
+    Fmt.epr "%s: expected %d, got %d\n%!" msg expected got;
+    failwith __LOC__)
+
+let check_string msg expected got =
+  if expected <> got then (
+    Fmt.epr "%s: expected %s, got %s\n%!" msg expected got;
+    failwith __LOC__)
+
+let check_kind msg expected got =
+  if
+    match (expected, got) with
+    | Inter.Val.Concrete.Node, `Node -> true
+    | (Contents | Contents_x _), `Contents -> true
+    | _ -> false
+  then (
```

Comment:
it looks like this should be the `else` clause, no?

Reply:
Yes, you're right! Thanks!